envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,20 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 requests.
+  // This is off by default, and not fully HTTP/1.0 compliant. There is planned support for HTTP/1.0
+  // style connect logic, dechunking, and handling lack of client host iff the default_host field is
+  // configured.
+  // Any requests for further 1.0 compatibility can be added `here
+  // <https://github.com/envoyproxy/envoy/issues/170`_
+  bool accept_http_10 = 2;
```

Comment:
folks still OK with accept_http_10 or can anyone think of something better than accept_http_10_and_09?